plugins/modules/aci_l2out.py

```diff
@@ -0,0 +1,322 @@
+     description: The original configuration from the APIC before the module has started
+     returned: info
+     type: list
+     sample:
+       [
+           {
+               "fvTenant": {
+                   "attributes": {
+                       "descr": "Production",
+                       "dn": "uni/tn-production",
+                       "name": "production",
+                       "nameAlias": "",
+                       "ownerKey": "",
+                       "ownerTag": ""
+                   }
+               }
+           }
+       ]
+   proposed:
+     description: The assembled configuration from the user-provided parameters
+     returned: info
+     type: dict
+     sample:
+       {
+           "fvTenant": {
+               "attributes": {
+                   "descr": "Production environment",
+                   "name": "production"
+               }
+           }
+       }
+   filter_string:
+     description: The filter string used for the request
+     returned: failure or debug
+     type: str
+     sample: ?rsp-prop-include=config-only
+   method:
+     description: The HTTP method used for the request to the APIC
+     returned: failure or debug
+     type: str
+     sample: POST
+   response:
+     description: The HTTP response from the APIC
+     returned: failure or debug
+     type: str
+     sample: OK (30 bytes)
+   status:
+     description: The HTTP status from the APIC
+     returned: failure or debug
+     type: int
+     sample: 200
+   url:
+     description: The HTTP url used for the request to the APIC
+     returned: failure or debug
+     type: str
+     sample: https://10.11.12.13/api/mo/uni/tn-production.json
+   '''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.cisco.aci.plugins.module_utils.aci import ACIModule, aci_argument_spec
+
+
+def main():
+    argument_spec = aci_argument_spec()
+    argument_spec.update(
+        bd=dict(type='str'),
+        l2out=dict(type='str', aliases=['name']),
+        domain=dict(type='str'),
+        vlan=dict(type='int'),
+        description=dict(type='str'),
+        target_dscp=dict(type='str'),
+        state=dict(type='str', default='present', choices=['absent', 'present', 'query']),
+        tenant=dict(type='str'),
+        name_alias=dict(type='str'),
+    )
+
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        required_if=[
+            ['state', 'absent', ['l2out', 'tenant']],
+            ['state', 'present', ['bd', 'l2out', 'tenant', 'domain', 'vlan']],
+        ],
+    )
+
+    bd = module.params['bd']
+    l2out = module.params['l2out']
+    description = module.params['description']
+    domain = module.params['domain']
+    vlan = module.params['vlan']
+    state = module.params['state']
+    tenant = module.params['tenant']
+    target_dscp = module.params['target_dscp']
+    child_classes = ['l2extRsEBd', 'l2extRsL2DomAtt', 'l2extLNodeP']
```

Comment:
move the child_classes just after the name_alias parameter so all parameters are together.